hel_django_auditlog_extra/tests/test_mixins.py

```diff
@@ -0,0 +1,194 @@
+from typing import List, Optional
+from unittest import mock
+
+from auditlog.context import set_actor
+from auditlog.models import LogEntry
+from auditlog.registry import auditlog
+from django.contrib import admin
+from django.contrib.admin.views.main import ChangeList
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
+from django.test import RequestFactory, TestCase
+
+from hel_django_auditlog_extra.mixins import AuditlogAdminViewAccessLogMixin
+from hel_django_auditlog_extra.tests.models import DummyTestModel
+
+User = get_user_model()
+
+REMOTE_ADDR = "127.0.0.1"
+
+
+class AuditlogAdminViewAccessLogMixinTest(TestCase):
```

Comment:
This was requested to be refactored by Kanslia's Ratkaisutoimisto, so that it is pure pytest and does not use Django TestCase.

- [ ] Refactor the Django TestCase to Pytest format.